tests/bug-5220/bug-5220-4/README.md

```diff
@@ -0,0 +1,10 @@
+# Test Description
+This test demonstrates a invalid use of fast_pattern along with base64_data
+where base64_data is succeeded by a buffer keyword and the content right
+after that is fast_pattern.
```

Comment:
README et rule do not seem to match

Reply:
Why?
`base64_decode:bytes 28; base64_data; bsize:=21; content:"something"; fast_pattern;`

> base64_data is succeeded by a buffer keyword and the content right
after that is fast_pattern

Reply:
What do you mean by buffer keyword ? I would expect `http.uri` or such

Reply:
I'm sorry. Because `bsize` is for buffer size, I called it a buffer keyword. Should have written payload keyword instead. Will fix. Thank you very much! 🙇🏽‍♀️